docs/data/stashcache/upgrades.md

```diff
@@ -0,0 +1,16 @@
+# OSG StashCache Upgrade Status
```

Comment:
I think this page belongs in the ops repo

Reply:
I would agree.  Though, one of the most common questions asked to the StashCache team is: "What are the specs of the caches?"

Reply:
Then let's link it from the cache install page